src/main/java/org/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -105,7 +105,10 @@
     UNDO("Undo", Localization.lang("Undo"), "ctrl+Z", KeyBindingCategory.EDIT),
     WEB_SEARCH("Web search", Localization.lang("Web search"), "alt+4", KeyBindingCategory.SEARCH),
     WRITE_XMP("Write XMP", Localization.lang("Write XMP"), "F6", KeyBindingCategory.TOOLS),
-    CLEAR_SEARCH("Clear search", Localization.lang("Clear search"), "ESCAPE", KeyBindingCategory.SEARCH);
+    CLEAR_SEARCH("Clear search", Localization.lang("Clear search"), "ESCAPE", KeyBindingCategory.SEARCH),
+    CLEAR_READ_STATUS("Clear read status", Localization.lang("Clear read status"), "alt+C", KeyBindingCategory.EDIT),
```

Comment:
I am not sure whether this is a good default.

Typically, in an application, there are shortcut keys for menu entries. They are typically used by <kbd>Alt<kbd> and then the letter. This is the same pattern than you propose. See also https://www.howtogeek.com/366147/how-to-make-windows-10-underline-and-highlight-menu-shortcut-keys/.

See for instance Excel:

![grafik](https://user-images.githubusercontent.com/1366654/103811689-99f3bb80-505d-11eb-81ab-efbfa38955e8.png)

We are trying to bring them back into JabRef.

Could you please think of other shortcuts? Maybe Ctrl+Alt+X?

Reply:
okay, how about:
- clear: ctrl+alt+x
- read: ctrl+alt+r
- skimmed: ctrl+alt+d   (ctrl+alt+s is "Save all")

Reply:
I think it would also be fine to have no default values.